static-site/src/util/parseMarkdown.js

```diff
@@ -50,8 +50,15 @@ export const parseMarkdown = (mdString) => {
     nonTextTags: ['style', 'script', 'textarea', 'option'],
     transformTags: {
       'a': function(tagName, attribs) { // eslint-disable-line
-        const url = new URL(attribs.href); // URL is not supported on Internet Explorer
-        if (url.hostname !== location.hostname) {
+        try {
+          const url = new URL(attribs.href); // URL is not supported on Internet Explorer
+          if (url.hostname !== location.hostname) {
+            attribs.target = '_blank';
+            attribs.rel = 'noreferrer nofollow';
+          }
+        } catch (err) {
+          // some valid-looking (and commonly used) URLs will cause `new URL` to
```

Comment:
@jameshadfield just for my own curiosity/learning, how did you figure out which URLs caused this? I had trouble logging from this function since its getting run inside the sanitize-html source code somewhere presumably.

Reply:
I guess it's easy enough to run it in a sandbox /separate context with some examples from Nextstrain Groups.

Reply:
There were helpful errors printed in the client console -- e.g. see the one in https://github.com/nextstrain/nextstrain.org/issues/361#issue-932134209.